clients/js/src/chains/sei/submit.ts

```diff
@@ -10,10 +13,13 @@ import { impossible, Payload } from "../../vaa";
 export const submit = async (
   payload: Payload,
   vaa: Buffer,
-  network: Network
+  network: Network,
+  rpc?: string
 ) => {
   const contracts = CONTRACTS[network].sei;
-  const { rpc, key } = NETWORKS[network].sei;
+  const networkInfo = NETWORKS[network].sei;
```

Comment:
we should be able to keep the line that was here and avoid the extra parameter if we update the networks.ts to include the rpc.

Reply:
agreed that using the public RPC would be nice. That said, there’s an argument to be made that all of these functions should propagate the RPC anyway to allow overriding for whatever reason

Reply:
I think it's worth supporting switching out the RPC url though :thinking: